### PR TITLE
Refactor p2p sample

### PR DESCRIPTION
**Pull request by grgustaf**
I think my main goal here was just to move the configurable part of the code up to the top of the file. :)

### CODE REVIEW

File: src/samples/.eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+// Copyright (C) <2018> Intel Corporation
```
Comment (jianjunz) [inline src/samples/.eslintrc.json:1]: <2020> since it's a newly created file.

Comment (grgustaf) [reply src/samples/.eslintrc.json]: Thanks, fixed that and also updated the new peercall.js header

**Comment by grgustaf** [comment]
@jianjunz do you think it would be reasonable to switch to Prettier? js-beautify seems to do some things that conflict with eslint, especially indentation.

**Comment by jianjunz** [comment]
Hi Geoff,

I found the js-beautifier issue as well. Recently, I'm using clang-format. It's okay to switch to another beautifier.

File: src/samples/p2p/js/peercall.js [hunk 2]
```diff
@@ -0,0 +1,176 @@
+// Copyright (C) <2018-2020> Intel Corporation
```
Comment (jianjunz) [inline src/samples/p2p/js/peercall.js:1]: We usually just write the year we create this file. So there is no need to update it later.

**Pull request merged**